_config.yml

```
@@ -33,11 +33,11 @@ aux_links:
   "GitHub <div src='assets/github.svg' class='social-button github'></div>":
     - "//github.com/tgc-utn"
   "Youtube <div class='social-button youtube'></div>":
-    - "//www.youtube.com/user/tgcentregas"
+    - "//youtube.com/user/tgcentregas"
   "Google group <div class='social-button gmail'></div>":
-    - "//groups.google.com/g/TecnicasDeGraficosPorComputadora/?authuser=4"
+    - "//groups.google.com/u/1/g/TecnicasDeGraficosPorComputadora"
```

mmmm, esto va a querer usar el segundo user que tenes logueado, 
```suggestion
    - "//groups.google.com/g/TecnicasDeGraficosPorComputadora"
```
si no me equivoco asi queda limpio, y que cada une elija cual usar